pkgs/top-level/all-packages.nix

```diff
@@ -13387,8 +13390,12 @@ let
 
   vym = callPackage ../applications/misc/vym { };
 
-  w3m = callPackage ../applications/networking/browsers/w3m {
+  w3m = callPackage ../applications/networking/browsers/w3m { };
+
+  w3m-pure = w3m.override {
     graphicsSupport = false;
+    x11Support = false;
+    mouseSupport = false;
```

Comment:
@oxij awesome work. 

i have only once comment just to keep things consistent with other derivations. usually we separate a package into <name> and <name>Full ... example (python and pythonFull). i think here we should do the same and have

w3m be what currently is w3m-pure and create new w3mFull which would bring all the graphicSupport, x11Support, mouseSupport. 

basically all i would like is to things be more or less consistent. @oxij you should wait with any changes until somebody else comments and confirms this.